src/main/java/AlexLink/HomeWork/HW7/City/City.java

```diff
@@ -0,0 +1,52 @@
+package AlexLink.HomeWork.HW7.City;
+
+public class City {
+    private String name;
```

Comment:
Do you not use inner class like fields? For example, our city contans list of streets and squares

Reply:
fixed it